exercises/proverb/example.py

```diff
@@ -1,7 +1,7 @@
-def proverb(itens, qualifier=''):
+def proverb(items):
```

Comment:
I like the name used in `proverb.py`. Can you change this to match for consistency?